src/main/java/io/debezium/connector/yugabytedb/YugabyteDBTablePoller.java

```diff
@@ -87,69 +92,142 @@ public void run() {
    * @return true if there is a new table in the stream info
    */
   private boolean areThereNewTablesInStream() {
-    try {
-      boolean shouldRestart = false;
-      GetDBStreamInfoResponse resp = this.ybClient.getDBStreamInfo(this.connectorConfig.streamId());
+    // Reset the retry counter.
```

Comment:
Comment doesn't add much value. You can add more information about why the reset is important here or simply remove the comment.